site/search/SearchPanel.tsx

```diff
@@ -281,22 +285,48 @@ const SearchResults = (props: SearchResultsProps) => {
                     const objectId = target.getAttribute(
                         "data-algolia-object-id"
                     )
-                    const position = target.getAttribute(
+
+                    const allVisibleHits = Array.from(
+                        document.querySelectorAll(
+                            ".search-results .ais-Hits-item a"
+                        )
+                    ).filter((e) => !isElementHidden(e))
+
+                    // starts from 1 at the top of the page
+                    const globalPosition = allVisibleHits.indexOf(target) + 1
+                    // starts from 1 in each section
+                    const positionInSection = target.getAttribute(
                         "data-algolia-position"
                     )
                     const index = target.getAttribute("data-algolia-index")
-                    if (objectId && position && index) {
+                    const href = target.getAttribute("href")
+                    const query = getWindowQueryParams().q
```

Comment:
Might be cleaner to pass the query (`this.inputValue`) in as a prop to `SearchResults`.